cmd/cloud/deployment.go

```diff
@@ -147,6 +148,7 @@ func newDeploymentCreateCmd(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVarP(&inputFile, "deployment-file", "", "", "Location of file containing the deployment to create. File can be in either JSON or YAML format.")
 	cmd.Flags().BoolVarP(&waitForStatus, "wait", "i", false, "Wait for the Deployment to become healthy before ending the command")
 	cmd.Flags().BoolVarP(&cleanOutput, "clean-output", "", false, "clean output to only include inspect yaml or json file in any situation.")
+	cmd.Flags().StringVarP(&deploymentEnforceCD, "enforce-cicd", "", "OFF", "Provide this flag either ON/OFF. ON means deploys to deployment must use an API Key or Token. This essentially forces Deploys to happen through CI/CD")
```

Comment:
flag with different default values should have different variables, i.e. we are using `deploymentEnforceCD` for both create and update commands, and we have different defaults in these two cases. But Cobra won't be able to differentiate it when parsing and populating variable values, so we can run into a situation where in `enforce-cicd` will have "OFF" as default even though not specified for update command.

Reply:
Are you recommending to use 2 different variables for this flag in create and update?

Reply:
yeah